src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1762,16 +1763,23 @@ constexpr bool has_return_tags_and_argument_tags_v =
  * db::item_type<ArgumentTags>..., Args...`
  * where the first two pack expansions are over the elements in the type lists
  * `MutateTags` and `ArgumentTags`, or have a static `apply` function that is
- * callable with the same types.
+ * callable with the same types. If you don't explicity specify the `MutateTags`
+ * and `ArgumentTags` template parameters, this function attempts to infer them
+ * from `F::return_tags` and `F::argument_tags`, respectively.
```

Comment:
I don't think this is correct.  The lists must be specified either in the functional or as template parameters, but not both.  (That is, you can't override the values in the functional by passing explicit parameters.)

Reply:
You are right, fixed the docs